### PR TITLE
Build for main branch

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -24,48 +24,48 @@ jobs:
...
+          # If you wish to specify custom queries, you can do so here or in a config file.
+          # By default, queries listed here will override any specified in a config file.
+          # Prefix the list here with "+" to use these queries and those in the config file.
+          # queries: ./path/to/local/query, your-org/your-repo/queries@main
 
-    # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
-    # If this step fails, then you should remove it and run the build manually (see below)
-    - name: Autobuild
-      uses: github/codeql-action/autobuild@v1
+      # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
```
Comment (mohitsuman) [inline .github/workflows/codeql-analysis.yml:54]: @sudhirverma Same space fix here.

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -24,48 +24,48 @@ jobs:
...
-    #- run: |
-    #   make bootstrap
-    #   make release
+      #- run: |
+      #   make bootstrap
+      #   make release
 
-    - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v1
+      - name: Perform CodeQL Analysis
```
Comment (mohitsuman) [inline .github/workflows/codeql-analysis.yml:70]: @sudhirverma Same space fix here.

File: pipeline-examples/output-pipelinerun.yaml [hunk 3]
```diff
@@ -5,10 +5,10 @@ metadata:
 spec:
   type: git
   params:
-  - name: revision
-    value: master
-  - name: url
-    value: https://github.com/GoogleContainerTools/skaffold
+    - name: revision
```
Comment (mohitsuman) [inline pipeline-examples/output-pipelinerun.yaml:8]: @sudhirverma Same space fix here.

File: pipeline-examples/output-pipelinerun.yaml [hunk 4]
```diff
@@ -61,27 +61,27 @@ metadata:
...
-        resource: source-repo
-  - name: then-check                 # 2.  check file exists
-    taskRef:
-      name: check-stuff-file-exists
-    resources:
-      inputs:
-      - name: workspace
-        resource: source-repo
-        from: [first-create-file]
+    - name: first-create-file # 1. create file
```
Comment (mohitsuman) [inline pipeline-examples/output-pipelinerun.yaml:67]: @sudhirverma It's mostly in the entire file. I don't think we need space fix here. Apart from extra spaces fix end of line. Please have a good look.

**Review by mohitsuman — CHANGES_REQUESTED** [review]

**Comment by mohitsuman** [comment]
This PR will fix #648 . @sudhirverma once merged, please rename the master to main to complete the issue fix.

**Review by mohitsuman — APPROVED** [review]

**Review by evidolob — APPROVED** [review]

**Pull request merged**